samples/snippets/detect_intent_stream.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DialogFlow API Detect Intent Python sample with audio files processed as an audio stream.
+
+Examples:
+  python detect_intent_stream.py -h
+  python detect_intent_stream.py --agent AGENT \
+  --session-id SESSION_ID --audio-file-path resources/hello.wav
+"""
+
+import argparse
+import uuid
+
+from google.cloud.dialogflowcx_v3beta1.services.sessions import SessionsClient
+from google.cloud.dialogflowcx_v3beta1.types import audio_config
+from google.cloud.dialogflowcx_v3beta1.types import session
+
+
+# [START dialogflow_detect_intent_stream]
+def detect_intent_stream(agent, session_id, audio_file_path, language_code):
+    """Returns the result of detect intent with streaming audio as input.
+
+    Using the same `session_id` between requests allows continuation
+    of the conversation."""
+    session_client = SessionsClient()
+    session_path = "{}/sessions/{}".format(agent, session_id)
```

Comment:
Please use f-strings in modern python versions:
```suggestion
    session_path = f"{agent}/sessions/{session_id}"
```

Reply:
(same for other places)